config/manifests/bases/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: openshift-storage
+  
+resources:
+- ocs-osd-deployer.clusterserviceversion.yaml
```

Comment:
Let's not assume downstream conditions and restrictions for upstream builds (which mean let's not hard code a namespace)

Reply:
Okay, I'll remove `namespace: openshift-storage` from kustomization.yaml but every time `make bundle` is executed it'll add `namespace: openshift-storage` to this file if the NAMESPACE parameter is not passed along with the command.
I mean to say that it's not hardcoded, kustomize add it to kustomization.yaml file when we build CSV and the namespace gets added depending on whatever is there in kustomization.yaml file.